src/operators.jl

```diff
@@ -595,6 +595,217 @@ function tensor_product(g::G, h::G) where {G<:AbstractGraph}
     return z
 end
 
+"""
+    strong_product(g, h)
+
+Return the [strong product](https://en.wikipedia.org/wiki/Strong_product_of_graphs)
+of `g` and `h`.
```

Comment:
I think it might be preferable to also document in a few sentences what the strong product is, so that somehow who checks the documentation in the REPL does not have to switch to wikipedia.

Reply:
Done.  I also added descriptions for the existing functions cartesian_product and tensor_product.